x-pack/legacy/plugins/apm/public/new-platform/plugin.tsx

```diff
@@ -135,21 +136,23 @@ export class ApmPlugin
 
     ReactDOM.render(
       <ApmPluginContext.Provider value={apmPluginContextValue}>
-        <i18nCore.Context>
-          <Router history={history}>
-            <LocationProvider>
-              <MatchedRouteProvider routes={routes}>
-                <UrlParamsProvider>
-                  <LoadingIndicatorProvider>
-                    <LicenseProvider>
-                      <App />
-                    </LicenseProvider>
-                  </LoadingIndicatorProvider>
-                </UrlParamsProvider>
-              </MatchedRouteProvider>
-            </LocationProvider>
-          </Router>
-        </i18nCore.Context>
+        <KibanaContextProvider services={{ ...core, ...plugins }}>
```

Comment:
Looks like this duplicates parts of the `apmPluginContextValue`. Might be a good follow-up task to remove `core` and `plugins` from that.

Reply:
Yeah - for the context here (for APM folks also) there is now a (currently undocumented, but can be documented) expectation that apps wishing to make use of things from the `Observability` plugin will use the official `<KibanaContextProvider />`. APM happened to be using a custom version of this already, but we needed to add this for the shared parts from `Observability`. 

Agreed, should be followed up on, but isn't a blocker.